test/contrib/test_autoguide.py

```diff
@@ -15,10 +15,14 @@
 from numpyro.distributions import constraints
 from numpyro.distributions.flows import InverseAutoregressiveTransform
 from numpyro.handlers import substitute
+from numpyro.infer_util import init_to_median
 from numpyro.svi import SVI, elbo
 from numpyro.util import fori_loop
 
 
+init_strategy = partial(init_to_median, num_samples=2)
```

Comment:
This reduces the time spent to find initial_params a bit. It took a few seconds to call `trace(model).get_trace(*args, **kwargs)` the first time.